static/app/views/organizationContext.tsx

```diff
@@ -105,8 +105,8 @@ export function OrganizationContextProvider({children}: Props) {
     if (organization && organization.slug === orgSlug) {
       return;
     }
-
     if (!orgSlug) {
+      OrganizationStore.setNoOrganization();
```

Comment:
This needs to be in a `useEffect`, otherwise you're creating a side effect during render.

Reply:
I don't understand this, the function that gets called directly below modifies the OrganizationStore (`fetchOrganizationDetails`)

Reply:
Oh sorry. you're right. I didn't expand the diff enough.